apps/server/src/modules/task/entity/submission.entity.ts

```diff
@@ -48,6 +48,14 @@ export class Submission extends BaseEntityWithTimestamps {
 	@ManyToMany({ fieldName: 'gradeFileIds', type: FileTaskInfo })
 	gradeFiles = new Collection<FileTaskInfo>(this);
 
+	isGraded(): boolean {
```

Comment:
i would use a getter for a calculaded property without outer dependencies... but thats taste...

Reply:
sorry i don't get "without outer dependencies". can you explain?

Reply:
i mean isGraded() is a calculated property that does not need anything from the outside of the entity. because of that i would use just a getter.

Reply:
I like the explizit way to declare that is a method and do stuff and not a property. If we use a key getter for it we disguise what it is.